csharp/Platform.IO/ConsoleHelpers.cs

```diff
@@ -38,9 +77,29 @@ public static string GetOrReadArgument(int index, string readMessage, params str
             }
         }
 
+        /// <summary>
+        /// <para>Output a input string to the console</para>
+        /// <para>Выводит переданныую строку</para>
+        /// </summary>
+        /// <param name="string">
+        /// <para>Text passed to the console</para>
+        /// <para>Текст переданный в консоль</para>
+        /// </param>
         [Conditional("DEBUG")]
         public static void Debug(string @string) => Console.WriteLine(@string);
 
+        /// <summary>
+        ///	<para>Output composit formatted text in the console</para>
+        /// <para>Выводит композитный форматированный текст в консоли</para>
+        /// </summary>
+        /// <param name="format">
+        /// <para>A composite format string</para>
+        /// <para>Строка составного формата</para>
+        /// </param>
+        /// <param name="args">
+        /// <para>Array of rest arguments</para>
+        /// <para>Массив остаточных аргументов</para>
```

Comment:
Look carefully in the [documentation about the args argument](https://docs.microsoft.com/en-us/dotnet/api/system.console.writeline?view=net-5.0#System_Console_WriteLine_System_String_System_Object___).